Sources/FoundationEssentials/JSON/JSONWriter.swift

```diff
@@ -131,134 +69,165 @@ internal struct JSONWriter {
     private let sortedKeys: Bool
     private let withoutEscapingSlashes: Bool
 
-    var data = Data()
+    var bytes = [UInt8]()
```

Comment:
reserve some space here. otherwise the first `{` is very expensive. I think we should always assume at least 256bytes of data? After that the reallocs will be very, very rare.